lists-service/cicd/deploy/pipeline/pipeline.yaml

```diff
@@ -0,0 +1,319 @@
+                   { "name":"ENVIRONMENT", "value":"${pEnvironment}" },
+                   { "name":"PIPELINE_FINGERPRINT", "value":"#{variables.PIPELINE_FINGERPRINT}" },
+                   { "name":"PRODUCT_COMPONENT", "value":"${pProductComponent}" },
+                   { "name":"SRC_BRANCH", "value":"${pGitHubBranch}" }
+                 ]
+            Namespace: ExportConfigNS
+            InputArtifacts:
+              - Name: 'SourceArtifact'
+            OutputArtifacts:
+              - Name: ExportConfigArtifact
+            RunOrder: 1
+
+          - Name: DeployLists
+            ActionTypeId:
+              Owner: AWS
+              Category: Build
+              Version: 1
+              Provider: CodeBuild
+            Configuration:
+              ProjectName: !Ref DeployLists
+              EnvironmentVariables: !Sub |
+                [
+                  { "name":"LISTS_SECRET_NAME", "value":"#{ExportConfigNS.LISTS_SECRET_NAME}" },
+                  { "name":"AWS_ACCOUNT_ID", "value":"${AWS::AccountId}" },
+                  { "name":"LISTS_ECR_REPOSITORY_NAME", "value":"#{ExportConfigNS.LISTS_ECR_REPOSITORY_NAME}" },
+                  { "name":"CLEAN_BRANCH", "value":"${pCleanBranch}" },
+                  { "name":"DB_READ_ENDPOINT", "value":"#{ExportConfigNS.DB_READ_ENDPOINT}" },
+                  { "name":"DB_WRITE_ENDPOINT", "value":"#{ExportConfigNS.DB_WRITE_ENDPOINT}" },
+                  { "name":"EKS_CLUSTER_NAME", "value":"${pEksClusterName}" }
+                ]
+            InputArtifacts:
+              - Name: 'SourceArtifact'
+            RunOrder: 2
+
+          - Name: DeployNotification
+            ActionTypeId:
+              Owner: AWS
+              Category: Build
+              Version: 1
+              Provider: CodeBuild
+            Configuration:
+              ProjectName: !Ref DeployNotification
+              EnvironmentVariables: !Sub |
+                [
+                  { "name":"AUTHOR", "value":"#{CheckoutSrcNS.AuthorDisplayName}" },
+                  { "name":"CLEAN_BRANCH", "value":"${pCleanBranch}" },
+                  { "name":"COMMIT_ID", "value":"#{CheckoutSrcNS.CommitId}" },
+                  { "name":"ENVIRONMENT", "value":"${pEnvironment}" },
+                  { "name":"PRODUCT_COMPONENT", "value":"#{ExportConfigNS.PRODUCT_COMPONENT}" },
+                  { "name":"PRODUCT_NAME", "value":"#{ExportConfigNS.PRODUCT_NAME}" },
+                  { "name":"REPO", "value":"#{CheckoutSrcNS.FullRepositoryName}" },
+                  { "name":"SLACK_ALERT_CHANNEL", "value":"#{ExportConfigNS.SLACK_ALERT_CHANNEL}" },
+                  { "name":"SLACK_DEPLOY_NOTIFICATION", "value":"#{ExportConfigNS.SLACK_DEPLOY_NOTIFICATION}" },
+                  { "name":"SRC_BRANCH", "value":"${pGitHubBranch}" }
+                ]
+            Namespace: DeployNotificationNS
+            InputArtifacts:
+              - Name: 'SourceArtifact'
+            RunOrder: 3
+
+      - Name: Teardown
```

Comment:
This needs an approval and action to teardown everything that was launched in the deploy action